### PR TITLE
not having a revision is bad ...

### PR DESCRIPTION
**Pull request by grosser**
@gabetax 

### Risks
 - None

### CODE REVIEW

**Comment by gabetax** [comment]
:+1:

**Comment by grosser** [comment]
added a bunch of other docker fixes, looks good ?

File: Dockerfile [hunk 1]
```diff
@@ -41,6 +36,9 @@ ADD config /app/config
 ADD app /app/app
 ADD lib /app/lib
 
+ADD REVISION /
```
Comment (grosser) [inline Dockerfile]: @jacobat do we need REVISION in the root for some reason / is just app good enough ?

Comment (gabetax) [inline Dockerfile]: ZDI uses it in the release task.

**Comment by gabetax** [comment]
:+1:

**Pull request merged**